pkg/kube/client.go

```diff
@@ -493,8 +496,18 @@ func (c *client) AllDiscoveryDo(ctx context.Context, istiodNamespace, path strin
 	for _, istiod := range istiods {
 		res, err := c.proxyGet(istiod.Name, istiod.Namespace, path, 15014).DoRaw(ctx)
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "Problem forwarding to %s.%s: %v; skipping.", istiod.Name, istiod.Namespace, err)
-			errs = multierror.Append(errs, err)
+			execRes, execErr := c.extractExecResult(istiod.Name, istiod.Namespace, discoveryContainer,
+				fmt.Sprintf("%s request GET %s", pilotDiscoveryPath, path))
+			if execErr != nil {
+				errs = multierror.Append(errs,
+					fmt.Errorf("error port-forwarding into %s: %v", istiod.Name, err),
```

Comment:
Any reason not to add  `istiod.Namespace` in the error message? I think it will be useful when the Istio is installed in a different namespaces.